pom.xml

```diff
@@ -10,7 +10,7 @@
   <artifactId>plugin</artifactId>
   <!-- Baseline Jenkins version you use to build and test the plugin. Users 
    must have this version or newer to run. -->
-  <version>2.8</version>
+  <version>1.609.3</version>
```

Comment:
I think that comment is not correct actually now.

can you try to keep 2.8 and use jenkins.version as 1.609.3?